buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -26,6 +26,9 @@ object Dependencies {
     const val workRuntimeKtx = "androidx.work:work-runtime-ktx:${Versions.Androidx.workRuntimeKtx}"
   }
 
+  object Protobufs {
```

Comment:
```suggestion
  object FhirProto {
```

the project name is FHIR Proto - so I'd be tempted to follow that.